### DISCUSSION TITLE
SAE und ZIGBEE

### DISCUSSION THREAD

**Discussion by kht1011** [camueller/SmartApplianceEnabler#509] [Allgemein]
Hallo,

Gibt es eine Möglichkeit, den SAE mit ZIGBEE Komponenten zu koppeln? Mit ZIGBEE Komponenten meine ich in erster Linie Schalter, aber möglicherweise auch andere Komponenten wie Lampen.

Gruß

**Comment by Trunks1982** [comment]
Wenn du iobroker oder so nutzt ja.
